src/scaict_uwu/service_container.py

```
@@ -0,0 +1,91 @@
+    """
+
+    _services_being_created: dict = {}
+    """
+    """
+
+    def load_wiring_files(self, wiring_modules: list) -> None:
+        for module_name in wiring_modules:
+            module = importlib.import_module(module_name)
+            self.apply_wiring(module.get_wiring())
+
+    def apply_wiring(self, service_instantiators) -> None:
```

Ruff (0.8.0)

Missing type annotation for function argument `service_instantiators`

(ANN001)